core/templates/dev/head/pages/skill_editor/skill_editor_navbar_directive.html

```diff
@@ -23,6 +23,10 @@
               &#xE2C3;
             </i>
             <span class="oppia-save-publish-button-label">Publish Changes</span>
+            <span class="hidden-sm hidden-xs" style="opacity:0.5"
+                  ng-if="getChangeListCount() > 0">
+              (<[getChangeListCount()]>)
```

Comment:
There is the same tag for "Save draft" above can we use one of this at the end instead of writing this twice. (Ignore if it's not possible)

Reply:
Done.